bin/build.sh

```diff
@@ -204,6 +206,13 @@ linux-@(ubuntu|neon)-@(16|18|20)*)
   fi
   ;;
 
+linux---x86_64)
+  BUILD_LLVM=1
+  BUILD_Z3=1
```

Comment:
I think the Z3 binary provided by its GitHub repo releases is general enough to run on AL2.

Reply:
I did not manage to get this to work. The version of GMP (or some other dependency, I forgot) that Z3 requires cannot be easily installed on AL2. So I could not get it to work. So I am building it from source so that it builds against the available older version of GMP.